apps/tangle-dapp/containers/NominatorStatsContainer/NominatorStatsContainer.tsx

```diff
@@ -86,6 +86,16 @@ const NominatorStatsContainer: FC = () => {
                 </Link>
               )}
 
+            <Link href={PagePath.ACCOUNT}>
+              <Button
+                variant="utility"
+                className="!min-w-[100px]"
+                isDisabled={isActiveAccountLoading}
```

Comment:
We should enable it even if there is no active account connected because the link should still function even without an account

Although I think I get why you chose to disable it, maybe because "View account" implies there's an active account?

Reply:
I only disable it when the account is being loaded, after that it can be null or have value but the button is still available

Reply:
![CleanShot 2024-04-10 at 16 37 23@2x](https://github.com/webb-tools/webb-dapp/assets/69841784/3dcc3ae3-0b01-422f-ab34-10dd54defc8f)